seq2science/schemas/config/gene_expression.schema.yaml

```diff
@@ -23,7 +23,7 @@ properties:
           quantifier_flags:
             default: '--seqBias --gcBias --validateMappings --recoverOrphans' # --posBias experimental flag
           decoy_aware_index:
-            default: False
+            default: False  # Requires ~65 GB RAM for GRCh38
```

Comment:
The memory could be added as a resource?

Reply:
It already is, but this is visible in the extended docs. Should I remove it?